doc/source/ray-air/getting-started.rst

```diff
@@ -43,159 +44,3 @@ AIR is for both data scientists and ML engineers alike.
   https://docs.google.com/drawings/d/1Qw_h457v921jWQkx63tmKAsOsJ-qemhwhCZvhkxWrWo/edit
 
 For data scientists, AIR can be used to scale individual workloads, and also end-to-end ML applications. For ML Engineers, AIR provides scalable platform abstractions that can be used to easily onboard and integrate tooling from the broader ML ecosystem.
-
-Quick Start
------------
-
-Below, we walk through how AIR's unified ML API enables scaling of end-to-end ML workflows, focusing on
-a few of the popular frameworks AIR integrates with (XGBoost, Pytorch, and Tensorflow).
-
-..
-  https://docs.google.com/drawings/d/1z0r_Yc7-0NAPVsP2jWUkLV2jHVHdcJHdt9uN1GDANSY/edit
-
```

Comment:
I removed the quick start since we still have quite a few end-to-end examples in the examples section, and this is somewhat redundant with the higher-level project quickstart